RELEASE_NOTES.md

```diff
@@ -1,3 +1,7 @@
+### Version 1.9.26 (September 24th, 2015)
+* Skipped 1.9.25
```

Comment:
Updated to match a previous entry like this. But it's not really any different. You got suggestions for this?